include/zmq.h

```diff
@@ -685,6 +685,7 @@ ZMQ_EXPORT void zmq_threadclose (void *thread_);
 /*  DRAFT ZMQ_RECONNECT_STOP options                                          */
 #define ZMQ_RECONNECT_STOP_CONN_REFUSED 0x1
 #define ZMQ_RECONNECT_STOP_HANDSHAKE_FAILED 0x2
+#define ZMQ_RECONNECT_STOP_AFTER_DISCONNECT 0x3
```

Comment:
this needs to go in src/zmq_draft.hpp as well

Reply:
ok, I will add it later.